src/components/axes/timeAxis.ts

```diff
@@ -278,15 +289,33 @@ export module Axis {
       return Math.min(this._getIntervalLength(config), this.width()) >= worstWidth;
     }
 
+    protected _getSize(availableWidth: number, availableHeight: number) {
+      // Makes sure that the size it requires is a multiple of tier sizes, such that
+      // we have no leftover tiers
+
+      var size = super._getSize(availableWidth, availableHeight);
+      size.height = this._tierHeights.reduce(function(prevValue, currValue, index, arr) {
+        return (prevValue + currValue > size.height) ? prevValue : (prevValue + currValue);
+      });
+      return size;
+    }
```

Comment:
This is exactly correct. `_getSize()` is used to retrieve the true size given a particular value of available space.